cristin-import/src/test/resources/features/GeneralMappingRules.feature

```diff
@@ -8,27 +8,26 @@ Feature: Mappings that hold for all types of Cristin Results
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has an additional identifier with key "Cristin" and value 12345
 
-  Scenario: Map returns NVA Resource with main title being the Cristin title annotated as
-  Original Title when there is only one CristinTitle and it is annotated as original
+  Scenario: NVA Resource gets as Main Title the single Cristin title which is annotated as
+  "Original Title" (i.e., the Cristin entry has no more titles except for the original title).
     Given the Cristin Result has an non null array of CristinTitles
     Given the Cristin Result has an array of CristinTitles with values:
       | Title Text         | Abstract Text                 | Status Original | Language Code |
       | This is some title | This is the original abstract | J               | en            |
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has an EntityDescription with mainTitle "This is some title"
 
-
-  Scenario: Map returns NVA Resource with main title being the Cristin title annotated as
-  Original Title when there are many titles but only one annotated as original
+  Scenario: NVA Resource gets as Main Title the only Cristin title annotated as Original Title when
+  there are many titles but only one annotated as Original.
     Given the Cristin Result has an array of CristinTitles with values:
       | Title Text                 | Abstract Text                 | Status Original | Language Code |
       | This is the original title | This is the original abstract | J               | en            |
       | This is translated title   | This is some other abstract   | N               | en            |
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has an EntityDescription with mainTitle "This is the original title"
 
-  Scenario: Map returns NVA Resource with Main Title being any Cristin Title annotated as
-  Original Title when there are two titles both annotated as original
+  Scenario: NVA Resource get as Main Title being any Cristin Title annotated as
+  Original Title when there are two titles both annotated as Original.
```

Comment:
When there are two titles both annotated as Original the NVA Resource gets any of the Cristin Titles annotated as
  Original Title as Main Title.